src/device/stm32/stm32f4-altfunc-bitfields.go

```diff
@@ -1,7 +1,6 @@
-// These are the supported alternate function numberings on the stm32f407
-// +build stm32,stm32f407
+// +build stm32f4
```

Comment:
Please add the following comments.

https://github.com/tinygo-org/tinygo/blob/26a0819119287713efe95f2ec0c091070c0bd3a3/src/device/stm32/stm32-moder-bitfields.go#L1-L3

Reply:
Agreed.

Reply:
Thanks @ardnew